src/config/colors.js

```diff
@@ -29,6 +29,8 @@ export const colors = {
   purple: '#8e44ad',
   orange: '#e67e22',
   githubDark: '#1f2327',
+  mercury: '#e3e3e3',
+  shark: '#24292e',
 };
```

Comment:
I know those are official names, but they're not explicite at all.
Also, do we really need to add news colors?

Reply:
As I understand it, we add all the colors to the configuration, or am I mistaken?

Reply:
yes, new colors should be added in configuration.
I was asking if we could nor reuse existing colors instead

Reply:
Okay, I remove those colors